src/evaluate.jl

```diff
@@ -1 +1,19 @@
-# Please fill this file
+"""
+t1: calculate maximum error from `t1` to `tf`
+"""
+function export_errors(file_path, t1::Real)
+    loaded_data = JLD2.load(file_path)
+    @unpack df, method, t0, tf, traj_des = loaded_data
+    @assert t1 < tf && t1 > t0
+    filtered_df = filter(:time => t -> t >= t1, df)
+    poss = filtered_df.sol |> Map(datum -> datum.plant.state.p) |> collect
+    poss_des = filtered_df.time |> Map(traj_des) |> collect
+    errors_pos = poss - poss_des |> Map(norm) |> collect
+    error_max = maximum(errors_pos)
+    # error("WIP")
+end
+
+function evaluate(errors, threshold)
+    @assert threshold > 0
+    mean(abs.((errors) .<= threshold))
+end
```

Comment:
```suggestion
"""
    evaluate(file_path, t1::Real, threshold::Real)

Evaluate whether the maximum error of trajectory data (loaded from `file_path`) does not exceed given `threshold` (check time period: `t1 < t < tf`).
"""
function evaluate(file_path, t1::Real, threshold::Real)
    @assert threshold > 0
    loaded_data = JLD2.load(file_path)
    @unpack df, method, t0, tf, traj_des = loaded_data
    @assert t1 < tf && t1 > t0
    filtered_df = filter(:time => t -> t >= t1, df)
    poss = filtered_df.sol |> Map(datum -> datum.plant.state.p) |> collect
    poss_des = filtered_df.time |> Map(traj_des) |> collect
    errors_pos = poss - poss_des |> Map(norm) |> collect
    maximum(errors_pos) <= threshold
end
```

보아하니 여러 개 중 뭐가 조건을 만족하는지의 비율을 따지는 것 같은데, 원래는 그냥 maximum 값이 threshold 를 넘으면 실패, 아니면 성공인거 아닌가요?
보고서 내용 참고 바랍니다.

Reply:
![image](https://user-images.githubusercontent.com/43136096/146648600-866d7a95-85f3-43a7-9e23-b07058f23626.png)

Reply:
음.. 여기에 바로 여러개의 trajectory에 따른 error들을 받아서 회복률을 계산하려고 헀었고, 현재는 한개의 error밖에 없어서 그렇습니다.

Reply:
생각해보니 이것은 밖으로 빼서 따로 계산하는게 좋겠네요.

Reply:
이미 작성한 내용이 있어서 일부만 가져다 쓰도록 하겠습니다 ㅎㅎ

Reply:
@JinraeKim 이건 별개의 얘기긴 한데 `df_filtered`와 같이 쓰는 것이 문법적으로 맞는 표현이라서 수정한 걸까요?..ㅎ

Reply:
문법은 상관없고, 그 객체의 속성을 나타내는 것을 앞으로 빼는 것이 보통 가독성이 좋습니다.
예:
`df`: dataframe
`df_filtered`: dataframe, filtered
...